vcd_cli/disk.py

```diff
@@ -96,13 +93,19 @@ def list_disks(ctx):
         disks = vdc.get_disks()
         result = []
         for disk in disks:
+            attached_vms = ''
+            if hasattr(disk, 'attached_vms') and \
+               hasattr(disk.attached_vms, 'VmReference'):
+                attached_vms = disk.attached_vms.VmReference.get('name')
```

Comment:
There can be multiple VmReference records, right?

Reply:
I think a VMDK can only be attached to a single VM at any given time, but the underlying API method returns a list of VMs. This is to be consistent with the API. http://pubs.vmware.com/vcd-820/index.jsp#com.vmware.vcloud.api.reference.doc_27_0/doc/types/VmsType.html?resultof=%2522%2561%2574%2574%2561%2563%2568%2565%2564%2576%256d%2573%2522%2520%2522%2561%2574%2574%2561%2563%2568%2565%2564%2576%256d%2522%2520